apps/marsha/vars/all/main.yml

```diff
@@ -37,3 +37,11 @@ marsha_should_sign_requests: true
 # -- volumes
 marsha_media_volume_size: 2Gi
 marsha_static_volume_size: 2Gi
+
+# -- resources requests
+marsha_resources_requests_app_cpu: 10m
```

Comment:
What about the following naming convention?

```
{{ app.name }}_{{ service}}_resources_cpu: 10m
```

Reply:
you're right the service should be first. As for `requests`, we need to keep it because resources limits exist alongside resources requests.

Reply:
Ok then.